### PR TITLE
add option for genome

### PR DESCRIPTION
**Pull request by senthil10**
in current RNA bp, the ref genome is always fetched from statusDB. Its convenient to have a parameter to pass in at times.

Its a small PR, so will great if anyone can merge it soon :smile:

### CODE REVIEW

File: scripts/RNA_analysis/make_tophat_sbatch.py [hunk 1]
```diff
@@ -189,24 +189,26 @@ def main(args,phred64,fai,projtag,mail,hours,conffile,fpath,single,stranded):
...
+    parser.add_option('-a', '--alloc-time', action="store", dest="hours", default="40:00:00",
     help="Time to allocate in SLURM. Please specify as hours:minutes:seconds or days-hours:minutes:seconds")
-    parser.add_option('-w', '--fpath', dest="fpath", default=False,     
+    parser.add_option('-w', '--fpath', dest="fpath", default=False,
     help="Path to fastq files. If not given, the script assumes the standars structure of the analysis directory'")
-    parser.add_option('-r', '--stranded', action="store_true", dest="stranded", default="False",     
+    parser.add_option('-r', '--stranded', action="store_true", dest="stranded", default="False",
     help="Run tophat with --librarytype fr-firststranded option for strand-specific RNAseq.")
+    parser.add_option('-g', '--genome', action="store", dest="genome", default=None,
+    help="Reference genome, if mentioned here not fetched from status DB")
```
Comment (ewels) [inline scripts/RNA_analysis/make_tophat_sbatch.py]: I find this slightly confusing.. Maybe:
`"Reference genome name (eg. 'GRCh37') Default: fetch from status DB"`

**Comment by ewels** [comment]
Looks good! Though that help message made me scratch my head a little..

**Comment by senthil10** [comment]
Haha, TRUE. Its friday evening man. I could not think much for the `help` :stuck_out_tongue:  Changed it now though :+1:

**Comment by guillermo-carrasco** [comment]
:+1:

**Pull request merged**